components/FilterItem.js

```diff
@@ -127,6 +127,9 @@ const FilterItem = forwardRef((props, ref) => {
 							source={require('../assets/spiral/thin.png')}
 							style={{ width: 80, height: 80 }}
 						/>
+						<StyledText className='absolute text-white text-[20px] font-bold w-[150px] text-center bottom-20'>
+							Search Circles
+						</StyledText>
```

Comment:
this title is really close to the circle. needs to be moved up. same with `view circles` title